sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -253,6 +253,7 @@ protected void doStart() {
 
     @Override
     protected void doStop() {
+        clusterService.remove(this);
```

Comment:
if this is required, the you should do this in the other services too which do add(this)